internal/stack/_static/docker-compose-stack.yml.tmpl

```diff
@@ -137,6 +137,9 @@ services:
     - type: bind
       source: ../../../tmp/service_logs/
       target: /tmp/service_logs/
+    - type: bind
```

Comment:
Could you add a comment here mentioning the systemd issue that made us to have this duplicated mount?